### PR TITLE
Add support for GPG signing

### PR DESCRIPTION
**Pull request by unguiculus**
Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

### CODE REVIEW

File: cr/cmd/package.go [hunk 1]
```diff
@@ -52,6 +55,15 @@ func getRequiredPackageArgs() []string {
...
 
 func init() {
+	dir, err := homedir.Dir()
+	if err != nil {
+		panic(err)
+	}
+
 	rootCmd.AddCommand(packageCmd)
 	packageCmd.Flags().StringP("package-path", "p", ".cr-release-packages", "Path to directory with chart packages")
+	packageCmd.Flags().Bool("sign", false, "Path to directory with chart packages")
```
Comment (davidkarlsen) [inline cr/cmd/package.go:65]: copy paste err in description?

**Comment by davidkarlsen** [comment]
lgtm except for the copy/paste description.

**Review by davidkarlsen — APPROVED** [review]

File: cr/cmd/package.go (hunk 1 above)
Comment (unguiculus) [reply cr/cmd/package.go:65]: ouch

**Comment by unguiculus** [comment]
Will be needed for Loki charts.
https://github.com/grafana/loki/issues/2593

File: cr/cmd/package.go (hunk 1 above)
Comment (unguiculus) [reply cr/cmd/package.go]: fixed

**Review by scottrigby — APPROVED** [review]
This looks great. For quite a while I've been wanting to make time to work on something to encourage signing charts 👏 👏

**Pull request merged**